src/main/java/com/minecolonies/achievements/MineColoniesAchievement.java

```diff
@@ -17,7 +17,7 @@
 {
 
     /**
-     * Inherit parent constructor
+     * Inherit parent constructor.
```

Comment:
that gives no value at all. Explain what this consturctor does. Document all parameters. (the same for the other constructors)